parts/header.html

```diff
@@ -0,0 +1,50 @@
+<!--
+wp:group {
+    "className": "site-header",
+    "layout": {
+        "type": "constrained"
+    },
+    "tagName": "header"
+}
+-->
+<header id="masthead" class="wp-block-group site-header">
+    <!--
+    wp:group {
+        "className": "site-header__inner",
+        "layout": {
+            "type": "flex",
+            "flexWrap": "nowrap"
+        }
+    }
+    -->
+    <div class="wp-block-group site-header__inner">
+        <!-- wp:site-logo /-->
+
+        <!-- create-wordpress-plugin:theme-navigation {"menuLocation": "header"} -->
```

Comment:
In progress: https://github.com/alleyinteractive/create-wordpress-project/issues/48